### DISCUSSION TITLE
Is there a way to give json/yaml/csv as input to scriggo command line

### DISCUSSION THREAD

**Discussion by newbeelearn** [open2b/scriggo#929] [Q&A]
I am able to see that we can provide "name=value" pair using const option as input to template but i don't see anyway to specify it in file.
It would be great if if json/yaml files can be given as input it would greatly enhance the utility of the tool. What do you think?

**Comment by gazerro** [comment]
I think it would be better to implement this functionality in a separate program. Implementing this in the scriggo command would add more complexity. The format of the JSON/YAML file should be documented and the next request would be to also define non-constants such as maps and slices, further increasing their complexity.

Considering that scriggo's `run` command is implemented in the file https://github.com/open2b/scriggo/blob/main/cmd/scriggo/run.go, it would be quite simple to implement with a separate command.
